modules/rosa-creating-cluster.adoc

```diff
@@ -20,6 +20,17 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 
 .Procedure
 
+. Enable ROSA to create JSON files for account-wide roles and policies, including Operator policies. This is supported for the `rosa init account` mode only:
```

Comment:
1. I am confused, the account-roles should be used for sts cluster, may I know why put it in regular rosa cluster creation doc?
2. It said `This is supported for the rosa init account mode only:` but the command next is `$ rosa create account-roles`
3. Is `manual` mode by default in command expected here for user to  run the commands by themselves? I think use `--mode auto -y` should be better in the command

Reply:
Sorry, I made the preceding comment when logged in with my personal GitHub account. I have now logged in to my Red Hat associated account.

Reply:
Yes, these steps are only for STS clusters. Good catch! All these changes should be moved to `rosa-sts-creating-cluster.html`

Reply:
Thanks for confirming that. I have relocated the content updates to https://deploy-preview-34960--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa-sts-creating-cluster.html.

Reply:
I have also removed the --mode option from the commands and included a description for each that refers to the interactive mode options. I have also included a note under each to state that it is possible to use the --mode option to specify the mode in the command directly.

Additionally, I have removed the "This is supported for the rosa init account mode only" line, following a conversation on this topic with Victor.